doc/index.rst

```diff
@@ -28,6 +28,18 @@ specify along with the Jupyter Notebooks inside. You will be able to share a URL
 with users that can immediately begin interacting with this environment via the
 cloud.
 
+What resources are given to Binder users?
+-----------------------------------------
+A Binder user is somebody that clicks on a Binder link in order to run a
```

Comment:
If you or another Binder user clicks on a Binder link, the ``mybinder.org`` deployment will run the linked repository. While running, you will be provided a minimum of 1G and a maximum of 4G of RAM.